streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -246,9 +246,7 @@ class StreamingContext private[streaming] (
     checkpointDir != null
   }
 
-  private[streaming] def initialCheckpoint: Checkpoint = {
-    if (isCheckpointPresent) _cp else null
-  }
+  private[streaming] def initialCheckpoint: Checkpoint = _cp
```

Comment:
@jaceklaskowski let's wrap this up or close it. Undo this change so that this is just about typos and it can be merged